mitodl/ocw-hugo-themes
# Adding Shira Segal to OCW Team listing

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1079.

#### What's this PR do?
Adds Shira Segal to OCW Team listing.

#### How should this be manually tested?
Run `yarn start www` and then navigate to `http://localhost:3000/about/`. Confirm that Shira Segal has been added to the team listing, and everything else looks correct.
